client/README.md

```diff
@@ -1,9 +1,15 @@
 # WIT-Calendar (client)
 
-## Building
-Run ``npm run build`` to build the extension. 
+## Setup
+1. Install ``npm`` on the node.js official website: https://nodejs.org/en/download
+2. Find where it says "Get a prebuilt Node.js for Windows running a x64 architecture"
```

Comment:
what if i'm on a mac?

Reply:
yea but what i'm saying is if I go to the page my button will look different then that, it will say mac not wiindows, and my arch type

Reply:
so it should be something more general here

Reply:
fair. its prob fine, unless @Cattn has any ideas or objections